src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+import javax.management.monitor.StringMonitorMBean;
+
+/**
+ * Base contact class.
+ * Handles value validation
+ */
+public class Contact {
+
+    public static final String CONSTRAINTS_MESSAGES = "None";
+    public static final String VALIDATION_REGEX = ".+";
+
+    public final String value;
```

Comment:
Can we prevent exposing of value outside of the class? In this case will `protected` scope help us achieve the same intended functionality?